index.html

```diff
@@ -51,6 +51,12 @@
   <body>
     <h1 id="header">Basics: Scissors Paper Stone 🚀</h1>
     <div id="container">
+      <p>This is a game of Scissors Paper Stone with two modes.</p>
+      <p><b>Regular game mode</b>: Type in "scissors", "paper" or "stone"</p>
+      <p>
+        <b>Reversed game mode</b>: Type in "reverse scissors", "reverse paper"
+        or "reverse stone"
+      </p>
```

Comment:
Good job in allowing both games to coexist together. I believe project 2 will challenge you to have different game modes, and then cater to the individual modes accordingly.